<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>619</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resouce.qrc">
    <normaloff>:/new/prefix1/resource/icon.jpg</normaloff>:/new/prefix1/resource/icon.jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton{
	background:transparent;
	border:none;
	icon-size: 50px 50px;
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="3,1">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="screenWidget" native="true">
      <layout class="QGridLayout" name="gridLayout_4">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="1" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>208</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_2" columnstretch="1,0">
         <item row="0" column="1">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="lbTitle">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>标题</string>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="4">
          <widget class="QPushButton" name="pbNext">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resouce.qrc">
             <normaloff>:/new/prefix1/resource/nextBlue.png</normaloff>:/new/prefix1/resource/nextBlue.png</iconset>
           </property>
          </widget>
         </item>
         <item row="1" column="5">
          <widget class="QComboBox" name="cbSpeed">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="focusPolicy">
            <enum>Qt::WheelFocus</enum>
           </property>
           <property name="acceptDrops">
            <bool>false</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">QComboBox {
	color: rgb(0, 170, 255);
    background-color: transparent;  /* 设置组合框背景颜色为透明 */
    border: none;         /* 设置组合框边框宽度、样式和颜色 */
}
QComboBox::drop-down {
  	border: none;
}</string>
           </property>
           <property name="editable">
            <bool>false</bool>
           </property>
           <property name="currentIndex">
            <number>1</number>
           </property>
           <property name="frame">
            <bool>true</bool>
           </property>
           <item>
            <property name="text">
             <string>0.5X</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1.0X</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2.0X</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="3" column="4">
          <widget class="QPushButton" name="pbOpen">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resouce.qrc">
             <normaloff>:/new/prefix1/resource/addBlue.png</normaloff>:/new/prefix1/resource/addBlue.png</iconset>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QPushButton" name="pbPlay">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resouce.qrc">
             <normaloff>:/new/prefix1/resource/playBlue.png</normaloff>:/new/prefix1/resource/playBlue.png</iconset>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QPushButton" name="pbStop">
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resouce.qrc">
             <normaloff>:/new/prefix1/resource/stopBlue.png</normaloff>:/new/prefix1/resource/stopBlue.png</iconset>
           </property>
          </widget>
         </item>
         <item row="3" column="1" colspan="3">
          <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
           <item>
            <widget class="QPushButton" name="pbVolume">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="resouce.qrc">
               <normaloff>:/new/prefix1/resource/volume.png</normaloff>:/new/prefix1/resource/volume.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="hsVolume">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="1" colspan="5">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QSlider" name="hsProgress">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lbTime">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <pointsize>12</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(255, 255, 255);</string>
             </property>
             <property name="text">
              <string>00:00:00/00:00:00</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="pbPrev">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resouce.qrc">
             <normaloff>:/new/prefix1/resource/prevBlue.png</normaloff>:/new/prefix1/resource/prevBlue.png</iconset>
           </property>
          </widget>
         </item>
         <item row="3" column="5">
          <widget class="QPushButton" name="pbList">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resouce.qrc">
             <normaloff>:/new/prefix1/resource/list.png</normaloff>:/new/prefix1/resource/list.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="lwMediaList">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>500</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QListWidget {
 background-color: rgb(0, 0, 0);
 color: rgb(255, 255, 255);
}
QListWidget::item:hover {
 background-color: #00aaff;
}

QListWidget::item:selected:focus {
 background-color: #0000ff;

}</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resouce.qrc"/>
 </resources>
 <connections/>
</ui>
